packages/openapi3/package.json

```diff
@@ -33,13 +33,15 @@
     "dist/**",
     "!dist/test/**"
   ],
-  "dependencies": {
+  "peerDependencies": {
     "@cadl-lang/compiler": "0.24.0",
     "@cadl-lang/rest": "0.7.0"
   },
   "devDependencies": {
     "@types/mocha": "~7.0.2",
     "@types/node": "~14.0.27",
+    "@cadl-lang/compiler": "0.24.0",
```

Comment:
why do these need to be repeated as devDependencies?

Reply:
this is an issue with rush that doesn't link peerDependencies. microsoft/rushstack#2807 talk about it. don't really agree with their view on the problem